### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by tanya-newStart**

### CODE REVIEW

File: javascript/javascript1/week1/age_calc.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const age = futureYear - yearOfBirth;
+
+console.log(`You will be ${age} years old in ${futureYear}.`);
+
+
+// A dog age calculator
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+
+const age_humanYears =dogYearFuture - dogYearOfBirth;
```
Comment (Rassibassi) [inline javascript/javascript1/week1/age_calc.js:14]: `age_humanYears` mixes `snake_case` with `camelCase`. It should be only `camelCase` meaning `ageHumanYears`.

Try to run prettier on this file. It is good practice to put spaces around operators, like `=`.

File: javascript/javascript1/week1/age_calc.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const dogYear = age_humanYears*7;
+
+let shouldShowResultInDogYears = false;
```
Comment (Rassibassi) [inline javascript/javascript1/week1/age_calc.js:17]: could it be a `const` ?

File: javascript/javascript1/week1/age_calc.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+if (shouldShowResultInDogYears){
+    console.log(`Your dog will be ${dogYear} dog years old in ${dogYearFuture}`);
+}
+else {
+    console.log(`Your dog will be ${age_humanYears} human years old in ${dogYearFuture}.`);
+};
```
Comment (Rassibassi) [inline javascript/javascript1/week1/age_calc.js:24]:
```javascript
if () {
    ...
} else {
    ...
};
```

File: javascript/javascript1/week1/house_price.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+function housePrice(width,depth,height, gardenSizeInM2){
+    const volumeInMeters = width*depth*height;
+    return volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+}
```
Comment (Rassibassi) [inline javascript/javascript1/week1/house_price.js:5]: add spaces after the commas in the function parameter list.

Also here add spaces between the operators in line `2`

You could name `volumeInMeters` to `volumeInM3` to be consistent.

File: javascript/javascript1/week1/house_price.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+if (housePrice(8,10,10,100)> 2500000){
```
Comment (Rassibassi) [inline javascript/javascript1/week1/house_price.js:6]: Spaces after the commas, and spaces around operators

File: javascript/javascript1/week1/name_generator.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    "Byte",
+    "Logic",
+    "Pulse",
+    "Edge"
+];
+
+const randomNumber = Math.floor(Math.random() * 10);//random number between 0 and 9
+const firstOne = firstWords[randomNumber];
+const secondOne = secondWords[randomNumber];
+const nameLength =firstOne.length+secondOne.length;
```
Comment (Rassibassi) [inline javascript/javascript1/week1/name_generator.js:29]: spaces around operators :)

File: javascript/javascript1/week1/name_generator.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+const firstWords =[
```
Comment (Rassibassi) [inline javascript/javascript1/week1/name_generator.js:1]: spaces around operators :)

**Review by Rassibassi — COMMENTED** [review]
Good job! Try to run prettier on your files :)

File: javascript/javascript1/week1/name_generator.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    "Insight",
+    "Byte",
+    "Logic",
+    "Pulse",
+    "Edge"
+];
+
+const randomNumber = Math.floor(Math.random() * 10);//random number between 0 and 9
+const firstOne = firstWords[randomNumber];
+const secondOne = secondWords[randomNumber];
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/name_generator.js:28]: `firstOne` and `secondOne` might not be the most descriptive variable names. Could `randomFirstWord` and `randomSecondWord` be better, or something else?

File: javascript/javascript1/week1/house_price.js (hunk 4 above)
Comment (jason-vasilev) [inline javascript/javascript1/week1/house_price.js:5]: That's a pleasant surprise to see you are already using functions. Bravo!

**Review by jason-vasilev — COMMENTED** [review]
You've understood well the key principles and have remembered well when to use `const` and `let`, with one exception. Also the use of `function` is nice to see. How could you solve the task without `functions`? You don't need to do it, but maybe explain in a comment.

File: javascript/javascript1/week1/name_generator.js (hunk 8 above)
Comment (tanya-newStart) [reply javascript/javascript1/week1/name_generator.js:28]: Yes, those are much better!

File: javascript/javascript1/week1/name_generator.js (hunk 7 above)
Comment (tanya-newStart) [reply javascript/javascript1/week1/name_generator.js:1]: got it! :)

**Pull request merged**